servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -2342,6 +2342,10 @@ RenderForwardMobile::RenderForwardMobile(RendererStorageRD *p_storage) :
 
 		scene_state.lightmap_buffer = RD::get_singleton()->storage_buffer_create(sizeof(LightmapData) * scene_state.max_lightmaps);
 	}
+	bool force_blinn = GLOBAL_GET("rendering/shading/overrides/force_blinn_over_ggx.mobile");
```

Comment:
`.mobile` here doesn't mean the mobile renderer, but it's a feature override to know if we're running on desktop or mobile. So users can use the mobile renderer, but have different values configured for `force_blinn_over_ggx` depending on the platform.

So you shouldn't check the value with the `.mobile` suffix (I'm not even sure that works). The `.<feature>` part is checked by the engine as an override and will be applied to any use of `"rendering/shading/overrides/force_blinn_over_ggx"` (including the clustered one if running on mobile).

Reply:
TY, didn't knew that